tests/docker/docker-compose.yml

```diff
@@ -1,4 +1,5 @@
 version: '3'
+name: s3_api
```

Comment:
i think i saw concerns with an underscore in another project...

perhaps... `irods-s3-api' is ... better?

Reply:
done